readme.txt

```diff
@@ -310,6 +310,8 @@ The plugin is made with love by [Modern Tribe Inc](http://m.tri.be/2s).
 
 = [4.6.11] TBD =
 
+* Fix - Don't modify the main query if is in a single post to allow custom variables don't affect the page loading when using shortcode [91787]
```

Comment:
Let's acknowledge one or more customers whenever we can :-)

Reply:
Joseph Reckley is one, so something like _"(our thanks to Joseph Reckley and others for highlighting this problem)"_ would work.